.github/workflows/release-please-libnpmaccess.yml

```diff
@@ -10,8 +10,13 @@ on:
       - release-next
       - latest
 
+permissions:
+  contents: read
```

Comment:
Not sure what the purpose of setting permissions that immediately get overwritten in a single job is.

Reply:
Removed it. Thanks